codewindtechpreview.md

```diff
@@ -0,0 +1,95 @@
+
+# Introducing Eclipse Codewind
+
+Welcome to a technical preview of Eclipse Codewind, an evolution of Microclimate!
+
+Our first Microclimate release was over a year ago now, and we've talked to customers about what's important to them and learned how to build tools for cloud native applications. As a result of this experience we've taken stock and decided to go in a new direction.
+
+Two of the major decisions we've made are to switch to the powerful [Eclipse Che](https://www.eclipse.org/che/) for our cloud-hosted IDE and evolve our CI/CD pipeline to be based on [Tekton Pipelines](https://tekton.dev). However, today is special for another reason - an initial preview of how we expect our support for local IDEs to evolve.
+
+![Microclimate evolution](dist/images/evolution.png "Microclimate evolution")
+
+Developers expect high-quality open source tools to be available in their IDE of choice. To deliver on this, we've started putting the local Microclimate installation on a diet and contributed it to a new Eclipse project that we've helped to form: Eclipse Codewind. Today we're happy to announce a first glimpse of what Codewind will look like. This preview shows the state of the codebase as it begins the move to Eclipse.
+
+Why Eclipse? That's easy - it's because the Eclipse Foundation has a strong history and community around building vendor-agnostic developer tools. Eclipse also hosts two of the main IDEs that we plan to extend, making it a natural fit for Codewind.
+
+The initial contribution for Codewind is coming from Microclimate. The Eclipse plug-in and Visual Studio Code extension are already open source and licensed under EPL 2. The common library that these depend on (a Docker container for now) has been relicensed under EPL 2 and is in the process of being legally cleared to release at Eclipse. All future releases of Codewind will come from the Eclipse codebase - we hope to see you there soon!
+
+## Limitations
+
+This preview is only available to use on Apple Mac (macOS).
+
+You must run a local installer (which installs almost 1 Gb of Docker containers - we're working on it!) in addition to the IDE plug-in/extension.
+
+Projects must be located in `<user>/microclimate-workspace`.
+
+## Prerequisites
+
+- Docker
+- Git
+- VS Code or Eclipse
+- Either uninstall Microclimate or prevent the Microclimate images from running. For more information about uninstalling Microclimate, see [Uninstalling Microclimate on Linux or Mac/OS](https://microclimate.dev/uninstalllinuxmac).
+
+## Installing Codewind
+
+#### Downloading the release binary on MacOS
+1. Download the installer binary file <a href="download/codewind" class="download-link trackdownload">codewind</a> to a folder on your system.
+2. If the binary file has the `.dms` extension, remove the extension so that the file is named `codewind`.
+3. Make the installer usable with the `chmod 775 codewind` command.
```

Comment:
since we call codewind to start/stop etc, maybe we don;t want to call it installer?